File: src/main/java/src/personas/Persona.java
package personas;

public abstract class Persona {
	//Atributos clase abstracta
	public String nombre;
	public int ID;
	public String Email;
	
	public Persona (String nombre, int ID, String Email) {
		this.nombre=nombre;
		this.ID = ID;
		this.Email=Email;
	}
	
	//metodos get y set 
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public int getID() {
		return ID;
	}
	public void setID(int ID) {
		this.ID=ID;
	}
	
	public String getEmail() {
		return Email;
	}
	public void setEmail(String Email) {
		this.Email=Email;
	}
}

File: src python/gestorAplicacion.personas/Persona.py

class Persona:
    def __init__(self, nombre, ID, Email):
        self.nombre = nombre
        self.ID = ID
        self.Email = Email

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getID(self):
        return self.ID

    def setID(self, ID):
        self.ID = ID

    def getEmail(self):
        return self.Email

    def setEmail(self, Email):
        self.Email = Email
